README.md

```diff
@@ -1,2 +1,61 @@
-# fire-data-processing
-Scripts to produce the fuel moisture and flammability data for the AFMS system
+##### Fire Data Processing Repository
+# ANU-WALD
+
+The Fire Data Processing repository is a collection of scripts used to produce the fuel moisture and flammability data for usage on the online interactive [Australian Flammability Monitoring system (AFMS)](http://wenfo.org/afms/).
+
+This is a research initiative to provide quality spatial information on fire hazards in Australia to better understand the risk of bushfires, their severity and preparing ways to respond to them.
+
+
+### Authors
+---
+##### Code
+*[Zac Hatfield Dodds](https://github.com/Zac-HD), [Dion Misic](https://github.com/kingdion)*
+
+##### Science
+*[Marta Yebra](https://researchers.anu.edu.au/researchers/yebra-m)*
+
+### Important Links
+---
+* [The Project](http://www.bnhcrc.com.au/research/understanding-mitigating-hazards/255)
+
+* [ANU Water and Landscape Dynamics Group](http://www.wenfo.org/wald/)
+
+* [Australian Flammability Monitoring System](http://wenfo.org/afms/)
+
+* [Australian Environment Explorer](http://wenfo.org/ausenv/)
+
+
+### Scripts
+---
+#### General Usage
+All scripts accept a help argument which displays all input flags. Use "python script.py --help" on the respective script to display help.
+
+You must be in the following NCI project groups to perform certain tasks with these scripts;
+
+* **ub8** - Data Access
+* **xc0** - To schedule Raijin jobs & edit scripts
```

Comment:
Add "Alternatively, you could edit the various `*.qsub` files to run under a different project's Raijin allocation."